### PR TITLE
New relic setup

### PR DESCRIPTION
**Pull request by elizabethcan**
This branch sets up New Relic and Artillery for stress testing. Currently the most I've been able to hit is around 600 RPS. I plan to look into ways to optimize my queries better.

### CODE REVIEW

File: data/index.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+const { Pool } = require('pg');
 
-const client = new Client({
+const pool = new Pool({
   // user: '',
   host: 'localhost',
   database: 'agent_contact',
   // password: '',
   // port: 3211,
 });
```
Comment (PlasmaIntec) [inline data/index.js:10]: What's your pool's max size? This parameter can be fine tuned to enhance your performance

File: newrelic.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['Zillions Agent Contact'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: '15c568520ba371053ccbef9bc68878224636dd99',
```
Comment (PlasmaIntec) [inline newrelic.js]: license_key should probably be left in a config file

**Comment by elizabethcan** [comment]
@PlasmaIntec can you re-review

**Review by PlasmaIntec — COMMENTED** [review]
Looks good @elizabethcan

**Pull request merged**